src/holofuel/pages/CreateRequest/CreateRequest.js

```diff
@@ -106,7 +109,11 @@ export function RenderNickname ({ agentId, setAgentNick }) {
        Loading
     </React.Fragment>
   }
-  if (error || !data.holofuelCounterparty.nickname) return <React.Fragment>No nickname available.</React.Fragment>
-  else setAgentNick(data.holofuelCounterparty.nickname)
+
+  if (error || !data.holofuelCounterparty.nickname) {
```

Comment:
this is a personal style preference of mine that I want to advocate goes in our code style guide. Specifically: 
always use the nested curly brace style of `if` statement when you include an `else` clause.
In fact, only use the one liner (no curly braces) style of `if` in the case where you are returning out of a function based on a condition. eg
`if (isLoading) return <>Loading...</>`

But no other case. We can discuss if you don't like this.

Reply:
Definitely a style preference, but I can grok that.  :)